packages/styles/src/buttons/buttons.scss

```diff
@@ -60,3 +60,35 @@
     font: $font-subheader;
   }
 }
+
+a.as-btn {
+  text-decoration: none;
+
+  &:hover {
+    background-color: var(--as-button-color-focus, #{$color-ui-02});
```

Comment:
This is something I want to talk later in the retrospective meeting, should we define these variables on top of the component so we know these variables are created here?

Reply:
yes, sure